### PR TITLE
Implement FileWrite

### PR DESCRIPTION
**Pull request by qti3e**

### CODE REVIEW

File: util.ts [hunk 1]
```diff
@@ -47,3 +47,12 @@ export function createResolvable<T>(): Resolvable<T> {
   });
   return Object.assign(promise, methods) as Resolvable<T>;
 }
+
+export function strToUint8Array(str: string): Uint8Array {
+  const ab = new ArrayBuffer(str.length);
+  const ta = new Uint8Array(ab);
+  ta.forEach((_, i) => {
```
[the remark this answers is not in the document]
Comment (qti3e) [reply util.ts]: @YurySolovyov thank you : )

**Comment by styfle** [comment]
I'm curious, why this is named `fileWrite` instead of `writeFile`?

**Comment by qti3e** [comment]
@styfle because it's going to have the same behaviour of good old `write` function, so we should call it `write` but then the name would be confusing so we should say it's `write` function from `file` family. (a group of functions).
So simply `File.Write -> FileWrite` : )
`man 2 write`

**Comment by styfle** [comment]
Go lang uses `ioutil.WriteFile` and Node.js uses `fs.writeFile` so I was expecting writeFile but I guess FileWrite is fine too 😅

**Comment by kevinkassimo** [comment]
Need to update `sendMsg` to `pubInternal`, see #136

**Comment by ry** [comment]
@qti3e we will revisit rust soon. Closing this now.

**Pull request closed without merging**